### DISCUSSION TITLE
Example System

### DISCUSSION THREAD

**Discussion by mcguirepr89** [mcguirepr89/BirdNET-Pi#50] [Show and tell]
I've made my system live again, and hope to keep it live and available to peruse as much as I can.

Take a look and hopefully you'll decide you'd like a BirdNET-Pi of your own!!

[BirdNET-Pi.pmcgui.xyz](https://birdnetpi.pmcgui.xyz)

**Comment by ghost** [comment]


**Reply by mcguirepr89** [reply]
I would love to hear what it can capture in the woods!

**Comment by DD4WH** [comment]
@mcguirepr89: Your system works very well and is really fast if you click around and listen to the extracted recordings.
My favourites are Purple Martin, Carolina Wren and of course the famous-by-TV Eastern Screech Owl :-)

**Reply by mcguirepr89** [reply]
Want to know a great secret??? That Purple Martin, Scarlet Tanager and Summer Tanager are actually all the Northern Mockingbird :)

There is also a Grey Catbird that likes to come around and he is often responsible for some other of my exotic, but not-really-there one-offs.

![image](https://user-images.githubusercontent.com/60325264/138932606-81c7d5ff-24e6-4faa-bd92-4281c7cebe31.png)
That Mockingbird makes to coolest Spectrograms

**Reply by DD4WH** [reply]
I have to learn a lot more about North American songbirds :-)

**Reply by mcguirepr89** [reply]
When the males get really desperate for a mate, sometimes they will sing for 10-15 minutes straight changing tunes every 2-5 seconds. They're one of my all-time favorites.

The other I love so much is my Norther Cardinal, who also happens to be the State Bird of North Carolina where I am from :)
![IMG_2052](https://user-images.githubusercontent.com/60325264/138933487-220af316-5f4e-4c1f-9352-ac548469919a.jpeg)

**Reply by DD4WH** [reply]
very nice!

**Comment by mcguirepr89** [comment]
Another example system! thanks to @Christoph-Lauer 

## [BirdNET-Pi in Germany](https://birdnetpigermany.hopto.org)

**Reply by ghost** [reply]
Thanks for all your work !!!

**Reply by mcguirepr89** [reply]
My pleasure! I'm working on the [Wiki for it all now. ](https://github.com/mcguirepr89/BirdNET-Pi/wiki/Sharing-Your-BirdNET-Pi)

**Reply by nilspupils** [reply]
Wow, thats quite a number of species!
